components/Header/Header.tsx

```diff
@@ -10,12 +10,13 @@ interface HeaderProps {
  * Displays the title and subtitle
  */
 const Header: React.FC<HeaderProps> = ({ title, subtitle }) => (
-  <div>
+  <div className="knowledge-panel-title">

```

Comment:
```suggestion
  <div>
```

Add  `knowledge-panel-title` (which I would rename to `header-title` but it doesn't really matter) to the `span` on line 14 that contains the title text, as currently you're applying the styles to the entire div instead of just the span (which doesn't hurt, but isn't accurate, and might affect the client if they're changing our styles).